Makefile

```diff
@@ -66,6 +66,10 @@ build.up: build.images up
 up:
 	@docker-compose up
 
+# Start the Docker daemon with docker-compose up.
+upd:
+	@docker-compose -d up
```

Comment:
We want to leave these running in the background so we need to start them as daemons.